src/perfume.ts

```diff
@@ -164,14 +172,14 @@ export default class Perfume {
     if (!this.checkMetricName(metricName)) {
       return;
     }
-    if (this.metrics.has(metricName)) {
+    if (this.metrics.hasOwnProperty(metricName)) {
```

Comment:
Based on https://jsperf.com/map-has-vs-object-hasownproperty results,
the most performant way would be simply to 
```js
if (this.metrics[metricName]) {
```

Reply:
good catch. Going to update this to use the fastest :) thx